[EDU-6321] fix: adjust Account policies guide (#1547)
aziontech/docs: src/content/docs/en/pages/guides/accounts/conditional-access-by-ip.mdx

  "active": true,
  "rules": [
    {
      "name": "My policy rule",
      "effect": "allow",
      "resource": "platform",

>       "actions": "access",

Reviewer: ```suggestion
      "actions": ["access"],
```

| Key | Type | Description |
| --- | ----------- | ----------- |
| name | String | Refers to the name of the conditional access policy. Up to *255 characters*. |
| active | Boolean | Indicates whether the policy is active. Accepted values: `true` and `false`. |
| rules | Array of objects | Includes an array of rules that define the policy's behavior. |
| name | String | In the rules' array, refers to the name of the individual rule. Up to *255 characters*. |
| effect | String | Refers to the rule's effect. Accepted values: `allow` and `deny`. |

> | resource | String | Specifies the resource or resources to which the rule applies, either as an exact URN or a pattern match. **Notes**: this endpoint includes the specific case of `platform` as a resource, which is exclusively used to define global actions for the platform. Accepted value: `.*` (matches all resources).|

Reviewer 2: ```suggestion
| resource | String | Specifies the resource or resources to which the rule applies, either as an exact URN or a pattern match. **Note**: this endpoint includes the specific case of `platform` as a resource, which is exclusively used to define global actions for the platform. Accepted value: `.*` (matches all resources).|
```

> | actions | String | Specifies the action or actions to which the rule applies. Accepted values: `create`, `retrieve`, `update`, and `destroy`. **Note**: when using `platform` as the resource, the only accepted action is `access`. |

Author: ```suggestion
| actions | Array of strings | Specifies the action or actions to which the rule applies. Accepted values: `create`, `retrieve`, `update`, and `destroy`. **Note**: when using `platform` as the resource, the only accepted action is `access`. |
```

> | actions | Array of strings | Specifies the action or actions to which the rule applies. Accepted values: `create`, `retrieve`, `update`, and `destroy`. **Note**: when using `platform` as the resource, the only accepted action is `access`. |

Reviewer 2: As palavras no mesmo estilo de `update`não estão em azul como costumam aparecer nestes casos. Revisar se está ok.

Author: na tabela, esse estilo não se diferencia. validado por aqui, todo ok

| condition | Text | Defines the conditions under which the rule applies. |
| ip_address | Array of strings | Includes an array of IP addresses that are evaluated by the condition. **Note**: mandatory field if using `platform` as the resource. |

2. You'll receive a response similar to the following:

```bash
{
  "state": "executed",
  "data": {
    "id": 0,
    "name": "My Policy",
    "last_editor": "my_username",
    "last_modified": "2025-02-24T14:15:22Z",
    "active": true,
    "rules": [
      {
        "name": "My policy Rule",
        "effect": "allow",
        "resource": "platform",

>         "actions": "access",

Reviewer: ```suggestion
        "actions": ["access"],
```

  "count": 123,
  "results": [
    {
      "id": 0,
      "name": "My policy",
      "last_editor": "my_username",
      "last_modified": "2025-02-24T14:15:22Z",
      "active": true,
      "rules": [
        {
          "name": "My policy rule",
          "effect": "allow",
          "resource": "platform",

>           "actions": "access",

Reviewer: ```suggestion
          "actions": ["access"],
```

}
```

3. Copy the `id` of the policy you want to update. 
4. Run the following `PUT` request in your terminal, replacing `[Token]` with your [personal token](/en/documentation/products/guides/personal-tokens/) and `<id>` with the identifier you copied in the previous step, to update the status of your policy. 

> This replaces all policy rules with the new data provided.

```bash
curl --request PUT \
  --url https://api.azion.com/v4/auth/policies/<id> \
  --header 'Accept: application/json' \
  --header 'Authorization: 123' \
  --header 'Content-Type: application/json' \
  --data '{
  "name": "My policy",
  "active": false,
  "rules": [
    {
      "name": "My policy rule",
      "effect": "allow",
      "resource": "platform",

>       "actions": "access",

Reviewer: ```suggestion
      "actions": ["access"],
```

{
  "state": "executed",
  "data": {
    "id": 0,
    "name": "My Policy",
    "last_editor": "my_username",
    "last_modified": "2025-02-24T14:15:22Z",
    "active": false,
    "rules": [
      {
        "name": "My policy Rule",
        "effect": "allow",
        "resource": "platform",

>         "actions": "access",

Reviewer: ```suggestion
        "actions": ["access"],
```

  "count": 123,
  "results": [
    {
      "id": 0,
      "name": "My policy",
      "last_editor": "my_username",
      "last_modified": "2025-02-24T14:15:22Z",
      "active": true,
      "rules": [
        {
          "name": "My policy rule",
          "effect": "allow",
          "resource": "platform",

>           "actions": "access",

Reviewer: ```suggestion
          "actions": ["access"],
```

{
  "state": "exectuted",
  "data": {
    "id": 0,
    "name": "My policy",
    "last_editor": "my_username",
    "last_modified": "2025-02-24T14:15:22Z",
    "active": true,
    "rules": [
      {
        "name": "My policy rule",
        "effect": "allow",
        "resource": "platform",

>         "actions": "access",

Reviewer: ```suggestion
        "actions": ["access"],
```
